lib/output.ts

```diff
@@ -69,10 +69,26 @@ export function mismatchingVersionsFixedToOutput(
     throw new Error('No fixes to output.');
   }
 
-  const dependencies = mismatchingDependencyVersions
-    .map((mismatchingVersion) => mismatchingVersion.dependency)
-    .sort();
-  return `Fixed versions for ${dependencies.length} ${
-    dependencies.length === 1 ? 'dependency' : 'dependencies'
-  }: ${dependencies.join(', ')}`;
+  const dependenciesAndFixedVersions = mismatchingDependencyVersions
+    .flatMap((mismatchingVersion) => {
+      let version;
+      try {
+        version = getHighestVersion(
+          mismatchingVersion.versions.map((v) => v.version)
+        );
+      } catch {
+        return []; // Ignore this dependency since unable to get the version that we would have fixed it to.
+      }
+      return {
+        dependency: mismatchingVersion.dependency,
+        version,
+      };
+    })
+    .sort((a, b) => a.dependency.localeCompare(b.dependency));
+
+  return `Fixed versions for ${dependenciesAndFixedVersions.length} ${
```

Comment:
`Fixed versions` seems vague. Can this be changed to something more descriptive?

Reply:
To me it's concise. Could also say "updated" instead of "fixed". Not sure it's a big difference. Let me know if you have suggestions.

Reply:
Maybe something along the lines of `Set versions to [...] across the workspace`.

Reply:
Well we need to have the list of dependencies at the end since it could be long and we don't want it in the middle of a sentence.